spamostack/keeper.py

```diff
@@ -119,6 +120,10 @@ def get(self, client_name, resource_name, param=None, func=None,
                     else:
                         probe = getattr(el, param)(*args, **kwargs)
 
+                    if (resource_name == "flavors"
```

Comment:
@FromZeus Hi, thank for review! I think that hardcoding a bunch of names is a wrong way :)
But, also i think that I can move `validate_uuid` method to `spam_factory` module, and iterate over list of flavors - and if id of flavor is uuid - we can remove this flavor. This allow us doesn't to break our cloud, and doesn't disrupt logic of `keeper.get()` method
Reason, why I using uuid validation - because standard flavors creates with custom id (not uuid) and our flavors will be created with auto-generated uuid